### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: veresone_theme/about.html [hunk 1]
```diff
@@ -213,17 +213,17 @@ <h1>Board of Governors </h1>
                             online, he was inducted into the inaugural class of
                             the Internet Society’s (ISOC) Internet Hall of Fame.
 
-                            Kilnam is currently Professor Emeritus of Korea
-                            Advanced Institute of Science and Technology
+                            Kilnam is currently Professor Emeritus of  Korea
```
Comment (davidlehn) [inline veresone_theme/about.html]: Remove added space.

File: veresone_theme/about.html [hunk 2]
```diff
@@ -242,7 +242,32 @@ <h1>Board of Governors </h1>
...
+                                alt="Anja Kovacs Picture">
+                        </div>
+                        <div style="font-size: 20px; font-weight: bolder; padding: 15px 0px;">Anja Kovacs</div>
+                        <p style="padding: 0px 20px;">
+                            Dr. Anja Kovacs directs the Internet Democracy
+                            Project in Delhi, India, which works towards an
+                            Internet that supports free speech, democracy and
+                            social justice in India and beyond. Anja’s research
+                            and advocacy focuses especially on questions
+                            regarding freedom of expression, cybersecurity and
```
Comment (davidlehn) [inline veresone_theme/about.html]: Using Oxford comma elsewhere, add one here.

**Comment by davidlehn** [comment]
Commit message has spelling typo and copyright wasn't actually updated.

**Review by msporny — CHANGES_REQUESTED** [review]
copyright is misspelled.

**Pull request merged**